### DISCUSSION TITLE
【求助】救命！我这个常驻池不对劲！

### DISCUSSION THREAD

**Discussion by Ailian-Qiao** [share121/inter-knot#393] [其他]
![Screenshot_20240806_003205](https://github.com/user-attachments/assets/54497042-04ac-435f-9005-edc09f6ac274)
我这个常驻池太不对劲了，不到50抽连着出三个S，我该怎么办，详情看图

**Comment by Vertinnnnn** [comment]
简单！将账号与密码交之吾手，吾帮你消了便可。（咬牙切齿）

**Comment by Ailian-Qiao** [comment]
账号77777
密码77777

**Comment by Vertinnnnn** [comment]
《治安局护航，防骗不用慌》

**Comment by SaKura-071721** [comment]
你不准玩zzz。q(≧▽≦q)

**Comment by xinxingjie** [comment]
这下好了吧，搞得大家都不开心了！

**Comment by gifttoword** [comment]
我比你适合玩zzzqaq

**Comment by Cangeltsiyu** [comment]
我**********（zzz脏话）

**Comment by MMHamburg** [comment]
可恶，这个委托我接了（今晚翻你家窗户
